tasks/clean/shutdown_vms.yml

```diff
@@ -1,8 +1,9 @@
-- name: Shutdown VMs
-  ovirt_vms:
-      state: stopped
-      name: "{{ vms.name }}"
-      force: True
-      wait: False
+# Get all the running VMs related to a storage domain and shut them down
+- name: Fetch VMs in the storage domain
+  ovirt_vms_facts:
+      pattern:  status != down and storage.name={{ storage['dr_' + dr_source_map + '_name'] }}
       auth: "{{ ovirt_auth }}"
 
+# TODO: Add a wait until the VM is really down
```

Comment:
What's this todo? You can just add `wait: true` to the `ovirt_vms` task

Reply:
Thanks will try that